src/languages/properties.js

```diff
@@ -0,0 +1,22 @@
+/*
+Language: Property file
+Contributors: Raffaele Sgarro <raffaeleSgarro@gmail.com>
+Category: config
+Description: Property files used in Java. A simple key value pair separated by '='
+*/
+function(hljs) {
+  return {
+    contains: [
+      hljs.HASH_COMMENT_MODE,
+      {
+        className: 'variable',
+        begin: /^[^=]+/,
```

Comment:
Keys and values can also be separated by the character `:`.
See https://docs.oracle.com/cd/E23095_01/Platform.93/ATGProgGuide/html/s0204propertiesfileformat01.html